book/managers_guide.md

```diff
@@ -0,0 +1,119 @@
+# Managing analytical code development
+
+This section is aimed at those who manage data analysis/science/engineering work in government or those acting as product owners for analytical products.
+
+Learning from previous projects, we've found that moving from traditional analysis approaches to developing analysis as code requires: 
+* Intention to transform and improve the quality of the wider business process around the project, not just automating an existing process. 
+* Committed, skilled resource for the duration of the project and for ongoing maintenance for the life span of the product. 
+* Motivation for the project and personal development from those working on the project. 
+* Having appropriate tools available for the development and deployment of the project. 
+* A plan for requirement collection and transitioning the product into business as usual. 
+
+It is common for this kind of transformation to identify quality issues in the current process. This should be seen as an opportunity to improve the quality of the process.
+
+[The Government Service Standard](https://www.gov.uk/service-manual/service-standard) outlines best practices for creating public services, which includes analysis. You should use this when designing and managing the development of analysis.
+
+The rest of this page describes the benefits of doing analysis as code and aims to help you ensure that your team are applying the good quality assurance practices outlined in the wider [Quality assurance of code for analysis and research guidance](https://best-practice-and-impact.github.io/qa-of-code-guidance/intro.html). Code that applies these good practices is referred to as a reproducible analytical pipeline (RAP).
+
+Analysis team managers should not need an in-depth understanding of the code produced by their team. However, they should be able to asses whether adequate quality assurance is being applied to the development of the analysis.
```

Comment:
I think there's a bit more to it. Suggest:
"Analysis team managers should not need an in-depth understanding of the code produced by their team. However, they should be confident that the approach the team has taken is appropriate given the user need and that adequate and proportionate quality assurance is being applied to the development and running of the analysis"